pages/api/replicache-pull.ts

```diff
@@ -28,7 +28,7 @@ type Cookie = z.TypeOf<typeof cookieSchema>;
 
 const pullRequestSchema = z.object({
   profileID: z.string(),
-  clientGroupID: z.string(),
+  clientGroupID: z.optional(z.string()),
```

Comment:
A better solution would be to look for `pullVersion: 1`.

You can use a type like:

```
const pullRequestV0Schema = z.object({
  pullVersion: z.literal(0),
  // other fields not used
});

const pullRequestV1Schema = z.object({
  pullVersion: z.literal(1),
  ... all the rest of current fields
});

const pullRequestSchema = z.union([pullRequestV0Schema, pullRequestV1Schema]);
type PullRequest = z.infer<typeof pullRequestSchema>();

Then you can validate that input matches `pullRequestSchema`. If it does check if it's version 0. If it is, send the error response you are now.

Reply:
See https://github.com/rocicorp/replicache-express/blob/main/src/backend/pull.ts for an example.